### PR TITLE
Cannot install jinja2 >=2.0,<=2.6 with setuptools >46,==3.0,==3.0.1,==3.0.2

### PR DESCRIPTION
**Pull request by mayaCostantini**
## Related Issues and Dependencies

Related to https://github.com/pypa/setuptools/issues/2017

## This introduces a breaking change

- No

## This should yield a new module release

- No

### CODE REVIEW

File: prescriptions/ji_/jinja2/setuptools46.yaml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+          name: setuptools
+          version: ">46,==3.0,==3.0.1,==3.0.2"
+          index_url: https://pypi.org/simple
+      - package_version:
+          name: jinja2
+          version: ">=2.0,<=2.6"
+          index_url: https://pypi.org/simple
+    run:
+      stack_info:
+      - &stack_info
```
Comment (fridex) [inline prescriptions/ji_/jinja2/setuptools46.yaml:19]: This anchor is not used:

```suggestion
      - 
```

**Review by fridex — COMMENTED** [review]
Nice! :+1:

File: prescriptions/ji_/jinja2/setuptools46.yaml (hunk 1 above)
Comment (mayaCostantini) [reply prescriptions/ji_/jinja2/setuptools46.yaml]: Thanks! :+1:

**Review by fridex — APPROVED** [review]
💯 👍🏻 🍪

**Pull request merged**